docs/rpc.md

```diff
@@ -207,6 +207,14 @@ the error-free C# response that provides a default result.
 NeoGo can generate an error in response to an invalid proof, unlike
 the error-free C# implementation.
 
+##### `getPeers`
+
+NeoGo extends the `getpeers` RPC call to return the user agent and
+last known block height for each connected peer where available.
```

Comment:
Probably we need to specify the exact field names, something like `to return the user agent (`useragent` JSON field)`, since it's the only place where this extension is documented.